tests/pdctl/store/store_test.go

```diff
@@ -357,4 +357,15 @@ func (s *storeTestSuite) TestStore(c *C) {
 	output, err = pdctl.ExecuteCommand(cmd, args...)
 	c.Assert(err, IsNil)
 	c.Assert(strings.Contains(string(output), "rate should less than"), IsTrue)
+
+	// store unsafe-recover
+	args = []string{"-u", pdAddr, "store", "unsafe-recover", "s1,s2,s3"}
```

Comment:
Do we also support store unsafe-recover?

Reply:
No, we do not, thanks for catching this. The test cases are remove.

We named the cmd to be `unsafe-recover` and under `store` parent cmd in the past, but changed it to the current name later. I forgot to delete this. Not sure why it did fail, maybe the `help` output is not considered as an error?